features/0036-issue-credential/README.md

```diff
@@ -35,9 +35,25 @@ There are two roles in this protocol: Issuer and Holder. Technically, the latter
 
 ### States
 
-The choreography diagrams shown below detail how state evolves in this protocol, in a "happy path."
+The choreography diagrams shown below detail how state evolves in this protocol, in a "happy path." The states include:
 
-Errors might occur in various places. For example, an Issuer might offer a credential for a price that the Holder is unwilling to pay. All errors are modeled with a `problem-report` message. Easy-to-anticipate errors reset the flow as shown in the diagrams, and use the code `issuance-abandoned`; more exotic errors (e.g., server crashed at Issuer headquarters in the middle of a workflow) may have different codes but still cause the flow to be abandoned in the same way.
+#### states for Issuer
+
+* proposal-received
+* offer-sent
+* request-received
+* credential-issued
+* done
+
+#### states for Holder
+
+* proposal-sent
+* offer-received
+* request-sent
+* credential-received
+* done
+
+Errors might occur in various places. For example, an Issuer might offer a credential for a price that the Holder is unwilling to pay. All errors are modeled with a `problem-report` message. Easy-to-anticipate errors reset the flow as shown in the diagrams, and use the code `issuance-abandoned`; more exotic errors (e.g., server crashed at Issuer headquarters in the middle of a workflow) may have different codes but still cause the flow to be abandoned in the same way. That is, in this version of the protocol, all errors cause the state of both parties (the sender and the receiver of the `problem-report`) to revert to `null` (meaning it is no longer engaged in the protocol at all). Future versions of the protocol may allow more granular choices (e.g., requesting and receiving a (re-)send of the `issue-credential` message if the Holder times out while waiting in the `request-sent` state).
```

Comment:
I see that list is the happy path set of states, but I think limiting it to just Happy Path states is not needed.  I think there should be a formally named non-happy path terminal state in the list - e.g. "issuance-abandoned". Or is there a reason not to make the list include such states?

To clarify, the lists should be all states we expect to use, and the reference to "happy path" in introducing the list should be changed.